tests/raftstore/test_stale_peer.rs

```diff
@@ -148,7 +155,7 @@ fn test_stale_peer_without_data<T: Simulator>(cluster: &mut Cluster<T>) {
     thread::sleep(Duration::from_secs(1));
 
     // check whether peer(2, 2) is destroyed
-    // if it's destroyed, it will write tomstone into the engine.
+    // if it's destroyed, it will write tombstone into the engine.
```

Comment:
if -> If.
What does the second "it" refer to? If it's already destroyed, can itself write "tombstone" into the engine?